connector/saml/saml.go

```diff
@@ -106,6 +104,9 @@ type Config struct {
 	//		urn:oasis:names:tc:SAML:2.0:nameid-format:persistent
 	//
 	NameIDPolicyFormat string `json:"nameIDPolicyFormat"`
+
+	// Allow a clock_drift when checking dates
+	AllowedClockDrift int64 `json:"allowedClockDrift"`
```

Comment:
In the saml-python implementation https://github.com/onelogin/python-saml/blob/master/src/onelogin/saml2/constants.py#L22 it is a constant value of 300 seconds.

In the saml-ruby implementation https://github.com/onelogin/ruby-saml/blob/15fb4789f8cd119fdbd722d58c659129d23a256d/lib/onelogin/ruby-saml/response.rb#L327-L329 it is not part of the settings but can be changed via options on the `Response class` and has a default value of 0 seconds.

Therefore I agree it should not be part of the configuration. I will remove it there but keep it as optional attribute in the provider struct. But which value should be used? 300 seconds seems pretty high to me. Should we go with the python value of 300 seconds?